playbooks/common/openshift-cluster/additional_config.yml

```diff
@@ -24,7 +24,3 @@
   - role: cockpit
     when: not openshift.common.is_atomic and ( deployment_type in ['atomic-enterprise','openshift-enterprise'] ) and
       (osm_use_cockpit | bool or osm_use_cockpit is undefined )
-  - role: flannel_register
```

Comment:
I think I would prefer to keep flannel_register in the 'Additional master configuration' play rather than having a separate 'Configure flannel' play. If it needs to run prior to any of the roles here, we should be able to just push it up in the order.